add_me_to_your_PATH/julia_pod

```diff
@@ -129,13 +130,8 @@ echo $JULIA_COMMAND
 # generate a k8s secret name
 SECRET_NAME=$(echo "${AWS_USER_ID}" | tr '[:upper:]' '[:lower:]' | sed -e 's/[^a-z0-9]\+/-/g')
 
-
-if kubectl get secret $SECRET_NAME &>/dev/null; then
-    kubectl delete secret $SECRET_NAME
-fi
-
```

Comment:
The point of this was not to cleanup the secret but ensure the secret is updated. Better option would be to use `kubectl replace` when the secret already exists.